src/main/java/graphql/language/TypeDefinition.java

```diff
@@ -14,4 +15,18 @@
      */
     List<Directive> getDirectives();
 
+    /**
```

Comment:
I am not sure about this: These methods do not add any real new value to the interface: they are just convenient.

If we really need/want an Interface with these methods we should add something like `DirectivesContainer` and make all classes implement is directly instead via `TypeDefinition`.